### PR TITLE
Tutorials update

### PR DESCRIPTION
**Pull request by JLBicho**
### Changelog

Updated tutorials *launch* and *params*

### Description

<table><tr><th>Before</th><th>After</th></tr><tr><td>

* launch does not include `foxglove-bridge`
* params does not include `waypoints` default value

</td><td>

* launch  includes `foxglove-bridge`
* params includes `waypoints` default value

</td></tr></table>

### CODE REVIEW

File: ros2/launch/launch/launch_example.launch.py [hunk 1]
```diff
@@ -26,19 +28,23 @@ def generate_launch_description():
...
+        cmd=["ros2", "launch", "foxglove_bridge", "foxglove_bridge_launch.xml"])
+
     # Add the nodes and the process to the LaunchDescription list
     ld = [compute_node,
           sensor_node,
           motor_node,
-          robot_node]
-          #foxglove_studio]
+          robot_node,
+          foxglove_studio,
```
Comment (james-rms) [inline ros2/launch/launch/launch_example.launch.py:47]: can we rename foxglove studio to foxglove while we're at it?

File: ros2/launch/launch/launch_example.launch.py [hunk 2]
```diff
@@ -26,19 +28,23 @@ def generate_launch_description():
...
-        parameters=[{"robot_name":"RobotA", 
-                     "max_speed":4.2, 
-                     "waypoints":["Home", "Room 1", "Corridor", "Home"]}]
+        parameters=[{"robot_name": "RobotA",
+                     "max_speed": 4.2,
+                     "waypoints": ["Home", "Room 1", "Corridor", "Home"]}]
     )
 
     # Launch Foxglove Studio to monitor data
     foxglove_studio = ExecuteProcess(cmd=["foxglove-studio"])
```
Comment (james-rms) [inline ros2/launch/launch/launch_example.launch.py:37]: @defunctzombie is our app binary still called `foxglove-studio` on linux?

File: ros2/diagnostics/package.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>diagnostics</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
```
Comment (james-rms) [inline ros2/diagnostics/package.xml:6]: fill in the TODOs?

File: ros2/diagnostics/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+rclcpp
+diagnostic_msgs
+std_msgs
+)
+
+add_executable(diagnostics_publisher src/main.cpp)
+ament_target_dependencies(diagnostics_publisher ${dependencies})
+
+install(TARGETS
+diagnostics_publisher
```
Comment (james-rms) [inline ros2/diagnostics/CMakeLists.txt:24]: indent?

**Review by james-rms — COMMENTED** [review]
A few nits but otherwise seems fine.

File: ros2/launch/launch/launch_example.launch.py (hunk 2 above)
Comment (defunctzombie) [reply ros2/launch/launch/launch_example.launch.py:37]: Yes

File: ros2/diagnostics/CMakeLists.txt [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(diagnostic_msgs REQUIRED)
+find_package(std_msgs REQUIRED)
+
+set(dependencies
+rclcpp
```
Comment (james-rms) [inline ros2/diagnostics/CMakeLists.txt:15]: another indent

**Review by james-rms — APPROVED** [review]

File: ros2/diagnostics_py/package.xml [hunk 6]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>diagnostics_py</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
```
Comment (james-rms) [inline ros2/diagnostics_py/package.xml:6]: another TODO here

File: ros2/diagnostics_py/package.xml [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  <maintainer email="jlmv.96@gmail.com">jose</maintainer>
+  <license>TODO: License declaration</license>
```
Comment (james-rms) [inline ros2/diagnostics_py/package.xml:8]: another TODO here

**Pull request merged**